ChangeLog.md

```diff
@@ -1,3 +1,25 @@
+## 2.1
+
++ Log naming and directory scheme has changed for both main keter logs and app logs.  
+  Old logs were named `dir/current.log` for the current log and `%Y%m%d_%H%M%S.log` 
+  (`time` package conventions) for rotated logs.  
+  Current logs have been brought up one level and named after their old directory:  
+  `logs/keter/current.log` -> `logs/keter.log`  
+  Rotated logs will now simply have `.1` `.2` ascending appended to the name of the base logs 
+  rather than be named after the date and time they were rotated at:  
+  `logs/keter/20230413_231415.log` -> `logs/keter/keter.log.1`  
+  Please update anything that depended on the old log naming and directory conventions accordingly.
```

Comment:
how does this work with several apps?

Reply:
Noticed a typo here actually :pray: Added some examples for app log changes just now.